Project/demo/src/main/java/com/example/demo/model/authenticationController.java

```diff
@@ -0,0 +1,37 @@
+    package com.example.demo.model;
```

Comment:
AuthenticationController should be in separate controller directory in order to follow MVC model